openslides_backend/action/mixins/import_mixins.py

```diff
@@ -43,8 +163,12 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
             raise ActionException(
                 f"Wrong id doesn't point on {self.import_name} import data."
             )
+        if worker.get("state") not in ImportState.__members__.values():
```

Comment:
```suggestion
        if worker.get("state") not in iter(ImportState):
```
should work as well. I would not use `__*__` methods if not absolutely necessary.

Reply:
Your proposal ends in a mypy error:

Reply:
That's unfortunate. Enums and mypy do not seem to get along very well... Then, I would suggest to either use `# type: ignore`/`cast(Iterator[str], iter(ImportState))` or use the following construct:
```
try:
    ImportState(worker.get("state"))
except ValueError:
    raise ActionException(...)
```
although I agree that that's barely better than accessing `__members__`...

Reply:
I think, the __members__ variant is the clearest and shortest and I want you to accept my opinion

Reply:
The mypy error is a bug in mypy: https://github.com/python/mypy/issues/16089. If you use e.g. `list(ImportState)`, it works without errors. I think that this is the clearest and most concise solution.

Reply:
Changed to `list(ImportState)`